closure/goog/testing/net/xhrio_test.js

```diff
@@ -55,6 +55,28 @@ function testStaticSend() {
   assertEquals('sendInstances_ after READY', 0, sendInstances.length);
 }
 
+function testStaticSendWithFactory() {
+  var xhrFactoryCalled = 0;
+  var optionsFactoryCalled = 0;
+
+  var wrapperFactory = new goog.net.WrapperXmlHttpFactory(
+      function() {
+        xhrFactoryCalled++;
+        return goog.net.XmlHttp();
```

Comment:
The use of `goog.net.XmlHttp` is incorrect here since this is a only the mock implementation of `goog.net.XhrIo`.
Historically it was the case that [`goog.net.XmlHttp` used to be the same as `MockXmlHttp`](https://github.com/google/closure-library/blob/dfe700923ca161b600cd7c057ece702841c18d51/closure/goog/net/xhrio_test.html#L81) in `goog.net.XhrIoTest` today.
Probably `MockXmlHttp` should be moved somewhere where `goog.testing.net.XhrIoTest` can use it too but since `goog.net.XmlHttp` is not globally mangled by the `MockXmlHttp` alias anymore this is all incorrect now and you either want to remove the use of `goog.net.XmlHttp();` or refactor as mentioned above.